packages/link/src/Link.js

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import Tooltip from "@crave/farmblocks-tooltip";
+import { fontSizes } from "@crave/farmblocks-theme";
+
+import linkTypes from "./constants/linkTypes";
+import Container from "./styledComponents/Link";
+
+export default class Link extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showTooltip: false
+    };
+
+    this.mouseOver = this.mouseOver.bind(this);
+    this.mouseLeaves = this.mouseLeaves.bind(this);
+  }
+
+  render() {
+    const { children, ...props } = this.props;
+    return (
+      <Container {...props}>
+        {props.leftIcon && <i className={`${props.leftIcon} margin-right }`} />}
+
+        {props.disabled ? (
+          <span onMouseOver={this.mouseOver} onMouseLeave={this.mouseLeaves}>
+            {children}
+          </span>
+        ) : (
+          <a href={props.href}>{children}</a>
+        )}
+
+        {props.external && <i className="wg-external-link margin-left" />}
+
+        <Tooltip
+          text="This action is disabled."
```

Comment:
can we turn this into a property? it would help with localization. Having hardcoded text is not a good idea, I have filed issue #135 to change this on the button  component as well